modules/chapter3/pages/section2.adoc

```diff
@@ -1,3 +1,150 @@
-= Section 2
+= Creating, Importing and Collaborating on Notebooks
+:navtitle: Git
 
-This is _Section 2_ of _Chapter 3_ in the *hello* quick course....
\ No newline at end of file
+In this section, you will learn how to create, import, and share notebooks by using a Git repository.
+
+The RHODS workbench images add the  https://github.com/jupyterlab/jupyterlab-git[Git] extension to JupyterLab.
+This extension adds graphical controls to interact with Git repositories via the JuypterLab interface.
+
+[NOTE]
+====
+If you prefer the command line, you can open a terminal and work directly with the `git` CLI.
```

Comment:
add the advantages of working with jupyter NB env, something like "but the JL NB env offers a,b,c....." at the end of this NOTE